syntax = "proto3";

package examples.clients.proto;

import "user.proto";

option java_outer_classname = "Product";

message ProductData {
  string product_id = 1;
  string name = 2;
  string description = 3;
  double price = 4;
  UserData seller = 5;
}
